### PR TITLE
User Story 61

### LINKED ISSUE #119: User Story 61, EXTENSION: Admin can manage items on behalf of a merchant
As an admin user
When I visit a merchant's profile page
I can click on the merchant's items link
And have access to all functionality the merchant does, including
- adding new items
- editing existing items
- enabling/disabling/deleting items

All content rules still apply (eg, item name cannot be blank, etc)

### PR DESCRIPTION
**Pull request by jfangonilo**
## What functionality does this accomplish?
closes #119 

**Description:**
Adds admin ability to add, edit, delete, and toggle status for items.
Refactors item forms

## What did you struggle on to complete?
Form for syntax

### Test Coverage Percentage: 100%
- [ ] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened):
- [x] New Tests added for new features

## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

## Please include a gif of how you feel about this branch:
tired

### CODE REVIEW

File: app/controllers/admin/items_controller.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
+class Admin::ItemsController < Admin::BaseController
```
Comment (wrenisit) [inline app/controllers/admin/items_controller.rb]: This is a big project! well done!

File: app/views/admin/items/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<h1>New Item for <%= @merchant.name %></h1>
+
+<%= form_for [:admin, @merchant, @item], url: admin_dash_merchant_items_path(@merchant) do |form| %>
```
Comment (wrenisit) [inline app/views/admin/items/new.html.erb]: Nice change here

File: app/views/items/_index.html.erb [hunk 3]
```diff
@@ -15,14 +16,17 @@
...
+          <%= link_to "Deactivate", "/merchant/items/#{item.id}/toggle_active", method: :patch if current_merchant_user? %>
+          <%= link_to "Deactivate", "/admin/items/#{item.id}/toggle_active", method: :patch if current_admin_user? %>
         <% else %>
-          <%= link_to "Activate", "/merchant/items/#{item.id}/toggle_active", method: :patch %>
+          <%= link_to "Activate", "/merchant/items/#{item.id}/toggle_active", method: :patch if current_merchant_user? %>
+          <%= link_to "Activate", "/admin/items/#{item.id}/toggle_active", method: :patch if current_admin_user? %>
         <% end %>
         <p><%= link_to "Delete Item", merchant_dash_item_path(item), method: :delete if item.no_orders? %></p>
-        <p><%= link_to "Edit Item", edit_merchant_dash_item_path(item)%> </p>
+        <p><%= link_to "Edit Item", edit_merchant_dash_item_path(item) if current_merchant_user? %> </p>
```
Comment (wrenisit) [inline app/views/items/_index.html.erb]: I like the change

**Review by wrenisit — APPROVED** [review]

**Review by rer7891 — APPROVED** [review]
Nice job with form_for

**Pull request merged**